Tools/LambdaTestTool-v2/src/Amazon.Lambda.TestTool/Commands/Settings/RunCommandSettings.cs

```diff
@@ -61,4 +61,11 @@ public sealed class RunCommandSettings : CommandSettings
     [CommandOption("--sqs-eventsource-config <CONFIG>")]
     [Description("The configuration for the SQS event source. The format of the config is a comma delimited key pairs. For example \"QueueUrl=<queue-url>,FunctionName=<function-name>,VisibilityTimeout=100\". Possible keys are: BatchSize, DisableMessageDelete, FunctionName, LambdaRuntimeApi, Profile, QueueUrl, Region, VisibilityTimeout")]
     public string? SQSEventSourceConfig { get; set; }
+
+    /// <summary>
+    /// The path used to save requests to.
+    /// </summary>
+    [CommandOption("--saved-requests-path <SAVED-REQUESTS-PATH>")]
+    [Description("The path used to save requests to.")]
+    public string? SavedRequestsPath { get; set; }

```

Comment:
do you think we should have a default value of saved requests to the current directory?

Reply:
i guess the other parameters like port we make them provide, so providing saved request path is fine then

Reply:
This is intentional. We don't want a default value for it.